README.md

```diff
@@ -53,31 +125,70 @@ Once you have stepped progress instances, you can interact with them using the m
 
   - `mySteppedProgress.getSteps()`: Get an array of the steps in the stepped progress component
   - `mySteppedProgress.getCompletedSteps()`: Get an array of the steps that are completed
-  - `mySteppedProgress.getCurrent()`: Get the step marked as current
+  - `mySteppedProgress.getCurrentStep()`: Get the step marked as current
   - `mySteppedProgress.isComplete()`: Get whether the stepped progress has been completed
   - `mySteppedProgress.progress()`: Mark the current step as complete, and move onto the next step
 
 There is [full API documentation available in the Origami Registry](https://registry.origami.ft.com/components/o-stepped-progress/jsdoc).
 
 ### Sass
 
-TODO
+#### Silent mode
```

Comment:
As we have a primary mixin, I wonder if we should start dropping silient mode documentention for new components?